satellite/satellitedb/accounting.go

```diff
@@ -20,6 +22,133 @@ type accountingDB struct {
 	db *dbx.DB
 }
 
+// ExceedsAlphaUsage returns true if more than 25GB of bandwidth or storage usage has been used in the past month
+// TODO: remove this code once we no longer need usage limiting for alpha release
+// Ref: https://storjlabs.atlassian.net/browse/V3-1274
+func (db *accountingDB) ExceedsAlphaUsage(ctx context.Context, projectID uuid.UUID) (bool, error) {
+
+	// check if limits have been exceeded in the past month
+	since := time.Now().AddDate(0, -30, 0)
+
+	exceeded, err := db.exceedsAlphaBandwidthUsage(ctx, projectID, since)
+	if err != nil {
+		return false, err
+	}
+	if exceeded {
+		return true, nil
+	}
+
+	exceeded, err = db.exceedsAlphaStorageUsage(ctx, projectID, since)
+	if err != nil {
+		return false, err
+	}
+	if exceeded {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+const (
+	// GB in bytes
+	gbBytes = (1 << (10 * 3)) // 1073741824 bytes
+	// For the alpha release, no more than 25GB of storage on disk or bandwidth usage can be exceeded
+	alphaMaxUsage = uint64(25 * gbBytes)
```

Comment:
This probably should be in a config flag somewhere, also unless you need a specific size for consts you can keep them unspecified, because they end up being easier used.